config/scim.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
 return [
+    "trace" => env('SCIM_TRACE',false),
```

Comment:
@uberbrady Does this do anything? I was hoping to use it too to try to debug some issues I'm running into with Okta.